escalate/core/views/model_tag.py

```diff
@@ -33,7 +33,7 @@ def post(self, request, *args, **kwargs):
             #above url not being redirected to in super.post so I brute-forced redirect
             return redirect(f'/{model_name}/{model_pk}')
```

Comment:
Tried changing success url when the cancel button was clicked, but when submit is clicked, the user is redirected back to the add tag page not the person edit page even though the success url changed. Same problem occurs when clicking submit. Hoping there are ideas on why this happens

Reply:
Interesting, I'll take a look at this after I accept this